articles/qdk-relnotes.md

```diff
@@ -19,8 +19,17 @@ For installation instructions, please refer to the [install guide](xref:microsof
 
 For update instructions, please refer to the [update guide](xref:microsoft.quantum.update-qdk).
 
-## Version 0.15.2103.133969
 
+## Version 0.16.xxxx.xxxxxx
+
+*Release date: April 22nd, 2021*
+
+- Improved Q# type inference based on the Hindley-Milner type inference algorithm.
+- [QEP 2: Enhanced Array Literals](https://github.com/microsoft/qsharp-language/blob/main/Approved/2-enhanced-array-literals.md) has been implemented.
+- Added support for more types for 'w' and 'indices' inputs for QIO Python SDK.
+- Updated control-plane CLI with new swagger to [support restricted access SKUs](https://github.com/Azure/azure-rest-api-specs/pull/13002).
```

Comment:
Also, recommend changing this to "Updated control-plane swagger file to [support ...](...)"
Let me prepare a separate PR for the CLI items to include past ones as well.